client/my-sites/hosting/main.js

```diff
@@ -72,10 +72,10 @@ class Hosting extends Component {
 
 		const getUpgradeBanner = () => (
 			<UpsellNudge
-				title={ translate( 'Upgrade to the Business plan to access all hosting features' ) }
+				title={ translate( 'Upgrade to the Pro plan to access all hosting features' ) }
```

Comment:
This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052